website/docs/r/volume.html.md

```diff
@@ -32,6 +33,8 @@ resource "hcloud_volume" "master" {
 - `size` - (Required, int) Size of the volume (in GB).
 - `server` - (Optional, int) Server to attach the Volume to, optional if location argument is passed.
 - `location` - (Optional, string) Location of the volume to create, optional if server_id argument is passed.
+- `automout` - (Optional, bool) Automount the volume after the attachment, server_id must be provider
```

Comment:
1. `automount`
2. Automount the volume upon attaching it (server_id must be provided).